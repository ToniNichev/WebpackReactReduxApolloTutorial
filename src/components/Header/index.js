import React from 'react';
import { Link } from 'react-router-dom';
const styles = require('./styles.scss');


const Header = ( {title} ) => (
  <div>
    <div className={styles.wrapper}>      
      <h2>{ title }</h2>
      <ul>
        <li><Link to='/page-one'>PAGE-ONE</Link></li>
        <li><Link to='/page-two'>PAGE-TWO</Link></li>
        <li><Link to='/page-three'>PAGE-THREEE</Link></li>        
      </ul>
    </div>
  </div>
);
export default Header;