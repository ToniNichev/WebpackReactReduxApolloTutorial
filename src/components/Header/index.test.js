// Link.react.test.js
import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';
import toJson from 'enzyme-to-json';

describe('Testing Header component', () => {
    it('renders as expected!!!', () => {
      const wrapper = shallow(
        <Header title="Title test" />
      );
     expect(toJson(wrapper)).toMatchSnapshot();
   });
});