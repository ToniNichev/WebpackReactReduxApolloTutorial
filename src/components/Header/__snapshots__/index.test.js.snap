// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component renders as expected!!! 1`] = `
<div>
  <div>
    <h2>
      Title test
    </h2>
    <ul>
      <li>
        <Link
          to="/page-one"
        >
          PAGE-ONE
        </Link>
      </li>
      <li>
        <Link
          to="/page-two"
        >
          PAGE-TWO
        </Link>
      </li>
      <li>
        <Link
          to="/page-three"
        >
          PAGE-THREEE
        </Link>
      </li>
    </ul>
  </div>
</div>
`;
